scripts/commit_checker.sh

```diff
@@ -0,0 +1,56 @@
+#!/bin/bash -e
+#
+# Copyright (C) 2020 HERE Europe B.V.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# SPDX-License-Identifier: Apache-2.0
+# License-Filename: LICENSE
+#
+
+echo "`git log --pretty=format:'%B' -2 | sed '1d' | sed '1d'`" >> commit.log
+cat commit.log
+for line in {1..15}
+do
+    current_line=`cat commit.log| sed -n ${line}p |wc -m | sed 's/\ \ \ \ \ \ /''/g' `
+    cat commit.log | sed -n ${line}p
+    if [ $line -eq 1 ] && [ ${current_line} -gt 50 ] ; then
+        echo "ERROR: Title is ${current_line} length, so it's too long for commit message title. It must be less than 50 chars "
```

Comment:
Can you please in addition to this error also print the full recommendation always:

```
Summarize changes in around 50 characters or less.
   
More detailed explanatory text. Wrap it to about 72
characters or so. In some contexts, the first line is treated as the
subject of the commit and the rest of the text as the body. The
blank line separating the summary from the body is critical (unless
you omit the body entirely); various tools like `log`, `shortlog`
and `rebase` can get confused if you run the two together.
   
Explain the problem that this commit is solving. Focus on why you
are making this change as opposed to how (the code explains that).
Are there side effects or other unintuitive consequences of this
change? Here's the place to explain them.
   
Further paragraphs come after blank lines.
   
- Bullet points are okay, too.
   
- Typically a hyphen or asterisk is used for the bullet, preceded
  by a single space, with blank lines in between, but conventions
  vary here.
   
All commits need to reference a ticket, again separated by a blank
line from the commit message summary above:
 
Resolves: ABC-111, #123
Relates-To: ABC-333, #321
See also: ABC-432, #456, #789
 
Signed-off-by: FirstName LastName <firstname.lastname@here.com>
```

Reply:
fixed.Added print of file